all.css

```diff
@@ -19,37 +29,443 @@ body {
   left: 52px;
   gap: 0px;
   opacity: 0px;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding: auto;
 }
-.navigation {
-  width: 401px;
-  height: 40px;
-  gap: var;
-  opacity: 0px;
+
+.navigation-list {
+  list-style-type: none;
+  display: flex;
 }
 
-.button {
-  text-decoration: none;
+.nav-link {
+  font-family: Poppins;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 24px;
   color: #f5f8fb;
+  text-decoration-line: none;
+}
+
+.button-discord {
+  font-family: Poppins;
   font-size: 18px;
-  position: relative;
-  border-width: 2px;
-  border-style: solid;
-  border-color: #d0d7fb;
-  width: 188px;
-  height: 48px;
-  box-sizing: border-box;
-  font-family: "Poppins", serif;
   font-weight: 600;
-  font-style: normal;
+  line-height: 28px;
+  text-align: left;
+  text-decoration-line: none;
+  color: #f5f8fb;
 }
-.button--blue {
+.button-connect {
   background-color: #0265dc;
   color: #f5f8fb;
   font-family: "Poppins", serif;
```

Comment:
Provide fallback fonts and family types at the end of the font list. For example: `font-family: "Roboto", Arial, sans-serif;`

Reply:
done